Modules/constants.c

```diff
@@ -104,6 +104,11 @@ LDAPerror(LDAP *l, char *msg)
             ldap_memfree(matched);
         }
 
+        if (pyctrls != NULL) {
+            PyDict_SetItemString(info, "ctrls", pyctrls);
+            /* Py_XDECREF(pyctrls) must be called on caller side */
```

Comment:
Needing to call `Py_XDECREF(pyctrls)` is implied by Python's calling convention – it's a borrowed reference. No need to point it out here.